### DISCUSSION TITLE
HA UI item Text message

### DISCUSSION THREAD

**Discussion by tdebans** [Seeed-Solution/SenseCAP_Indicator_ESP32#55] [Ideas]
Hi,

Idea to ad a UI template for text.
So it is possible to ad a text in a HA Text Helper -> pas it true mqtt -> show it on the SenseCAP.

Idea is to send a message to someone at the front door.

**Comment by Love4yzp** [comment]
> I have an idea to add a UI template for text. This would make it possible to add text in a Home Assistant Text Helper, pass it through MQTT, and show it on the SenseCAP.
>
>The idea is to send a message to someone at the front door.

You mean that right?
Yes, it is easy to modify it to have that function.

**Reply by tdebans** [reply]
yes that is what i mean.

**Reply by Love4yzp** [reply]
Expect to see your customization.
